tests/Integration/inc/Engine/Common/PerformanceHints/Cron/Subscriber/cleanup.php

```diff
@@ -1,16 +1,16 @@
 <?php
 declare(strict_types=1);
 
-namespace WP_Rocket\Tests\Integration\inc\Engine\Media\AboveTheFold\Cron\Subscriber;
+namespace WP_Rocket\Tests\Integration\inc\Engine\Common\PerformanceHints\Cron\Subscriber;
 
 use WP_Rocket\Tests\Integration\TestCase;
 
 /**
- * Test class covering \WP_Rocket\Engine\Media\AboveTheFold\Cron\Subscriber::atf_cleanup
+ * Test class covering \WP_Rocket\Engine\Common\PerformanceHints\Cron\Subscriber::cleanup
  *
- * @group AboveTheFold
+ * @group PerformanceHints
  */
-class Test_AtfCleanup extends TestCase {
+class Test_Cleanup extends TestCase {
 	protected $path_to_test_data = '/inc/Engine/Media/AboveTheFold/Cron/Subscriber/atfCleanup.php';
```

Comment:
```suggestion
	protected $path_to_test_data = '/inc/Engine/Common/PerformanceHints/Cron/Subscriber/atfCleanup.php';
```
You need to update the fixture location as well.